SpatialGDK/Source/SpatialGDK/Private/Interop/SpatialReceiver.cpp

```diff
@@ -349,6 +369,13 @@ void USpatialReceiver::OnAuthorityChange(const Worker_AuthorityChangeOp& Op)
 		return;
 	}
 
+	StaticComponentView->OnAuthorityChange(Op);
```

Comment:
Could you add a quick comment about why it's necessary to do it here and not in `HandleActorAuthority`

Reply:
Added a comment here, and also a comment here as to why we buffer authority if we are in a critical section as I think that is the less intuitive part of this function